infrastructure/tari_script/src/op_codes.rs

```diff
@@ -623,18 +621,15 @@ mod test {
     fn check_height() {
         fn test_check_height(op: &Opcode, val: u8, display: &str) {
             // Serialize
-            assert!(matches!(
-                Opcode::read_next(&[val, 1, 2, 3]),
-                Err(ScriptError::InvalidData)
-            ));
-            let s = &[val, 63, 0, 0, 0, 0, 0, 0, 0, 1, 2, 3];
+            assert!(matches!(Opcode::read_next(&[val, 255]), Err(ScriptError::InvalidData)));
+            let s = &[val, 63, 1, 2, 3];
             let (opcode, rem) = Opcode::read_next(s).unwrap();
             assert_eq!(opcode, *op);
             assert_eq!(rem, &[1, 2, 3]);
             // Deserialise
             let mut arr = vec![1, 2, 3];
             op.to_bytes(&mut arr);
-            assert_eq!(&arr, &[1, 2, 3, val, 63, 0, 0, 0, 0, 0, 0, 0]);
```

Comment:
This is weird that this passes. The op code is in the middle of the value. I was expecting 63 to occur before the value

Reply:
my bad, 63 is the height

Reply:
The existing test is confusing, 63 is the height value and `val` is the op code.
Should use the constants in tests and pass in a `val` and `expected_opcode` into `test_check_height`